posthog/admin/tests/test_redis_mutation_admin.py

```diff
@@ -0,0 +1,324 @@
+import pytest
+from django.contrib.admin.sites import AdminSite
+from django.contrib.admin.utils import flatten_fieldsets
+from django.test import RequestFactory
+from django.test.client import Client
+from rest_framework import status
+
+from posthog.admin.admins.redis_mutation_admin import RedisMutationAdmin
+from posthog.api.test.test_organization import create_organization
+from posthog.models import User
+from posthog.models.redis import RedisMutation
+from posthog.redis import get_client
+
+
+@pytest.fixture
+def request_factory():
+    return RequestFactory()
+
+
+@pytest.fixture
+def admin():
+    admin = RedisMutationAdmin(model=RedisMutation, admin_site=AdminSite())
+    return admin
+
+
+@pytest.fixture
+def superuser():
+    organization = create_organization("Test Org")
+    superuser = User.objects.create_and_join(organization, "test@posthog.com", "abcde12345", is_staff=True)
+    return superuser
```

Comment:
The django `admin_user` fixture wasn't really working for me, and I had this one written already.